tests/test_burst.py

```diff
@@ -168,3 +168,28 @@ def test_get_burst_params_multiple_results():
         with pytest.raises(ValueError, match=r'.*found multiple results.*'):
             burst.get_burst_params('there are multiple copies of this burst')
         mock_search.assert_called_once_with(product_list=['there are multiple copies of this burst'])
+
+
+def test_validate_bursts():
+    try:
+        burst.validate_bursts(  # valid burst pair
+            'S1_030349_IW1_20230808T171601_VV_4A37-BURST',
+            'S1_030349_IW1_20230820T171602_VV_5AC3-BURST'
+        )
+    except ValueError as e:
+        assert False, f'Unexpected exception for valid burst pair: {e}'
```

Comment:
No need to try/except here, the intention is clear when reading the code and the test will fail if an exception is raised by the function.
```suggestion
    burst.validate_bursts(
        'S1_030349_IW1_20230808T171601_VV_4A37-BURST',
        'S1_030349_IW1_20230820T171602_VV_5AC3-BURST'
    )
```